README.md

```diff
@@ -81,6 +81,14 @@ Profiling using eBPF incurs lower overhead & provides kernel stacks. This (curre
 **By default, gProfiler sends logs to Granulate Performance Studio** (when using `--upload-results`/`-u` flag)
 This behavior can be disabled by passing `--dont-send-logs` or the setting environment variable `GPROFILER_DONT_SEND_LOGS=1`.
 
+### Metrics and metadata collection
```

Comment:
Also mention that if `-u` is not given, nothing is uploaded (and ensure it's actually so :sweat_smile: )